package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "hathor-wallet-service",
+  "version": "1.4.3+beta",
```

Comment:
question: Do we really use `+` on the beta versions? Or should we have `1.4.3-beta`?

Reply:
It's because in the release guide we use `-rc` for release candidate versions so we agreed upon using `+` for the beta, e.g. `v1.4.3-rc1+beta`

Reply:
We really use it. Mostly because we don't want it to be considered as a pre-release (as in https://semver.org/#spec-item-9), as just metadata (as in https://semver.org/#spec-item-10).

This makes it easier for some tools in our pipeline to detect the release-candidates, which are our actual pre-releases, and not mistake releases with pre-releases because of the `-beta`.

Reply:
Thanks for the reference to the semver docs! It's pretty clear to me now why the `+`